### ISSUE TITLE
Can not build Wekan v4.44 for Sandstorm with Meteor 2.0-beta3 and Node 14.15.0 and fibers

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#3316]
@kentonv @zenhack

Hi,
it seems I can not build Wekan, because I can't get new version of fibers to correct directory. There are errors like fibers.node or fibers binary missing.

I have added those broken files to https://releases.wekan.team/broken/ , changes to meteor-spk are node v14 binary and added fibers. Although, probably I should add that with v14 built fibers to Wekan bundle, but I can't figure out how to get Sandstorm to detect paths correctly.
```
** Starting Meteor...
## There is an issue with `node-fibers` ##
`/programs/server/node_modules/fibers/bin/linux-x64-83-glibc/fibers.node` is missing.

Try running this to fix the issue: node /programs/server/node_modules/fibers/build
Error: Cannot find module '/programs/server/node_modules/fibers/bin/linux-x64-83-glibc/fibers'
Require stack:
- /programs/server/node_modules/fibers/fibers.js
- /programs/server/boot.js
- /main.js
- /start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:14:39)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
/programs/server/node_modules/fibers/fibers.js:23
		throw new Error('Missing binary. See message above.');
		^

Error: Missing binary. See message above.
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:23:9)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/programs/server/boot.js:2:15)
** HTTP-BRIDGE: App server exited with status code: 1
```
When I added fibers to meteor-spk deps, then it complained about different path:
```
** SANDSTORM SUPERVISOR: Starting up grain. Sandbox type: userns
** Starting Mongo...
2020-10-28T20:39:45.075+0000 I STORAGE  Engine custom option: log=(prealloc=false,file_max=200KB)
about to fork child process, waiting until server is ready for connections.
forked process: 11
child process started successfully, parent exiting
** Starting Meteor...
## There is an issue with `node-fibers` ##
`/programs/server/node_modules/fibers/bin/linux-x64-83-glibc/fibers.node` is missing.

Try running this to fix the issue: node /programs/server/node_modules/fibers/build
Error: Cannot find module '/programs/server/node_modules/fibers/bin/linux-x64-83-glibc/fibers'
Require stack:
- /programs/server/node_modules/fibers/fibers.js
- /programs/server/boot.js
- /main.js
- /start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:14:39)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
/programs/server/node_modules/fibers/fibers.js:23
		throw new Error('Missing binary. See message above.');
		^

Error: Missing binary. See message above.
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:23:9)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/programs/server/boot.js:2:15)
** HTTP-BRIDGE: App server exited with status code: 1
** SANDSTORM SUPERVISOR: App exited with status code: 1
** SANDSTORM SUPERVISOR: Starting up grain. Sandbox type: userns
** Starting Mongo...
2020-10-28T20:39:46.976+0000 I STORAGE  Engine custom option: log=(prealloc=false,file_max=200KB)
about to fork child process, waiting until server is ready for connections.
forked process: 11
child process started successfully, parent exiting
```

**Comment by xet7** [comment]
Well, meanwhile I released Wekan v4.44 for Sandstorm using Node 12.19.0, that does work. I'll try to figure out Node 14.15.0 later.

**Comment by zenhack** [comment]
Fwiw, Sandstorm itself is still on node 12, so I'm not surprised you had trouble.
